addons/base_vat/models/res_partner.py

```diff
@@ -249,7 +249,7 @@ def check_vat_hu(self, vat):
         if individual:
             return True
         # Check the vat number
-        check_func = stdnum.util.get_cc_module('hu', 'vat').is_valid(vat)
+        check_func = getattr(stdnum.util.get_cc_module('hu', 'vat'), 'is_valid', None)
         if check_func is not None:
             return check_func(vat)
```

Comment:
Don't use `getattr` to make the CI/security happy :wink: 
```suggestion
        return stdnum.util.get_cc_module('hu', 'vat').is_valid(vat)
```